spark/src/test/scala/geotrellis/spark/io/hadoop/HadoopSpaceTimeRasterCatalogSpec.scala

```diff
@@ -0,0 +1,66 @@
+package geotrellis.spark.io.hadoop
+
+import java.io.IOException
+
+import geotrellis.raster._
+import geotrellis.vector._
+
+import geotrellis.spark._
+import geotrellis.spark.ingest._
+import geotrellis.spark.io._
+import geotrellis.spark.io.index._
+import geotrellis.spark.tiling._
+import geotrellis.raster.op.local._
+import geotrellis.proj4.LatLng
+import geotrellis.spark.testfiles._
+
+import org.scalatest._
+import org.apache.hadoop.fs.Path
+import com.github.nscala_time.time.Imports._
+
+class HadoopSpaceTimeRasterCatalogSpec extends FunSpec
```

Comment:
Usually we have Spec class names line up with actual class names. So these would maybe belong in a `describe` block in `HadoopRasterCatalogSpec`.